src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Autentiseringa har feila.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Påloggingsinformasjonen vart ikkje funnen.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <target>Autentiseringa vart ikkje prosessert på grunn av ein systemfeil.</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>Invalid credentials.</source>
+                <target>Ugyldig påloggingsinformasjon.</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>Cookie has already been used by someone else.</source>
+                <target>Cookie-en er allereie brukt av ein annan brukar.</target>
```

Comment:
This would probably involve diverging from `no`, but should probably be `Informasjonskapsel`, or not applicable in nn?

Reply:
It's allowed. I'll review, same as above.

Reply:
"Kjeks" is also allowed here, actually :)

Reply:
Ha.. Lets skip the crackers :)